pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -297,6 +298,77 @@ func (r *ReconcileStorageCluster) newStorageClasses(initData *ocsv1.StorageClust
 	return ret, nil
 }
 
+// newRBDSnapshotClass returns a new RBD backed SnapshotClass
+func (r *ReconcileStorageCluster) newRBDSnapshotClass(initData *ocsv1.StorageCluster) *snapapi.VolumeSnapshotClass {
+	retSC := &snapapi.VolumeSnapshotClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: generateNameForRBDSnapshotClass(initData),
+		},
+		Driver: generateNameForRBDSnapshotClassDriver(initData),
+		Parameters: map[string]string{
+			"clusterID": initData.Namespace,
+			"csi.storage.k8s.io/snapshotter-secret-name":      "rook-csi-rbd-provisioner",
+			"csi.storage.k8s.io/snapshotter-secret-namespace": initData.Namespace,
+		},
+		DeletionPolicy: snapapi.VolumeSnapshotContentDelete,
+	}
+	return retSC
+}
+
+func (r *ReconcileStorageCluster) newSnapshotClasses(instance *ocsv1.StorageCluster) []*snapapi.VolumeSnapshotClass {
+	scs := []*snapapi.VolumeSnapshotClass{
+		r.newRBDSnapshotClass(instance),
+		// TODO
```

Comment:
TODO need to be fixed.

Reply:
@Madhu-1 , @humblec , do we have the `cephfs` snapshot module completed? Can some one point me to the latest changes?

Reply:
https://github.com/ceph/ceph-csi/pull/394 for cephfs which is planned for release v3.1.0 of ceph-csi

Reply:
Removed the `TODO` and added the cephfs snapshotter.
@Madhu-1 , @humblec , please take a look.